### DISCUSSION TITLE
Documentation / example of svelte app with adjusting the configuration

### DISCUSSION THREAD

**Discussion by maku** [nxext/nx-extensions#119] [Ideas]
Is there a documentation or example how to do configuration of a typical svelte app based on rollup? (e.g. for usage of tailwind, postcss, routify)

**Comment by DominikPieper** [comment]
Hey, sorry, right now, there's no such documentation. Docs are fairly minimal yet, but I plan to do some work here the next time. With the knowledge from version 1, I'll change some things here and, after that, hopefully, deploy some example repos and more docs. Sorry about that!
